server/commonTransactions/createCells/steps.ts

```diff
@@ -6,6 +7,83 @@ import CellDoorNumbers from './cellDoorNumbers'
 import SetCellType from '../setCellType'
 import Capacities from './capacities'
 import modifyFieldName from '../../helpers/field/modifyFieldName'
+import RemoveCellType from '../removeCellType'
+
+function wrapSetCellTypeController(path: string, step: FormWizard.Step) {
+  if (path === '/set-cell-type/:cellId') {
+    return class WrappedSetCellTypeController extends step.controller {
+      override successHandler(req: FormWizard.Request, res: Response, next: NextFunction) {
+        const pathPrefix = req.form.options.fullPath.replace(/\/set-cell-type\/.*/, '')
+        const history = req.journeyModel.get('history') as FormWizard.HistoryStep[]
+
+        this.addJourneyHistoryStep(req, res, {
+          path: history.find(item => {
+            return item.next.endsWith('/capacities')
+          }).next,
+          next: `${pathPrefix}/set-cell-type/:cellId/type`,
+          wizard: req.form.options.name,
+          revalidate: false,
+          skip: false,
+          editing: req.isEditing && !req.notRevalidated ? true : undefined,
+          continueOnEdit: req.isEditing && !req.notRevalidated ? true : undefined,
+        })
+        super.successHandler(req, res, next)
+      }
+    }
+  }
+  if (path === '/remove-cell-type/:cellId') {
+    return class WrappedRemoveCellTypeController extends step.controller {
+      override successHandler(req: FormWizard.Request, res: Response, next: NextFunction) {
+        const pathPrefix = req.form.options.fullPath.replace(/\/remove-cell-type\/.*/, '')
+        const history = req.journeyModel.get('history') as FormWizard.HistoryStep[]
+
+        this.addJourneyHistoryStep(req, res, {
+          path: history.find(item => {
+            return item.next.endsWith('/capacities')
+          }).next,
+          next: `${pathPrefix}/capacities`,
+          wizard: req.form.options.name,
+          revalidate: false,
+          skip: true,
+          editing: req.isEditing && !req.notRevalidated ? true : undefined,
+          continueOnEdit: req.isEditing && !req.notRevalidated ? true : undefined,
+        })
+        super.successHandler(req, res, next)
+      }
+    }
+  }
```

Comment:
I don't think we need to alter the history here because we have entryPoint: true in the remove steps.ts